resharper/resharper-unity/src/ProjectModel/MetaFileTracker.cs

```diff
@@ -210,7 +206,13 @@ private void OnItemRemoved(ProjectItemChange change)
                     RenameMetaFile(metaFile, newMetaFile, " via add/remove");
                 }
                 else
+                {
+                    // Only delete the meta file if the original file or folder is missing
+                    if (change.OldLocation.Exists != FileSystemPath.Existence.Missing)
+                        return;
+                    
```

Comment:
We can just get rid of this whole check. The old file will always still exist AFAICT, because we remove it from the project model, send out the notifications, and then remove from disk.

Leaving this check in means that a safe delete action will fail - we've removed the file from the project model, we get the notification, the file still exists on the disk and so we fail to delete the `.meta` file.

Reply:
So that would also fix #1231

Reply:
So that would also fix #1231